src/main/java/com/kunmi/taskManager/service/project/Project.java

```diff
@@ -0,0 +1,31 @@
+package com.kunmi.taskManager.service.project;
+
+import lombok.*;
+
+import java.time.LocalDateTime;
+import java.util.concurrent.atomic.AtomicInteger;
+
+@Getter
+@Setter
+@ToString
+@AllArgsConstructor
+@EqualsAndHashCode
+public class Project {
+
+    private static final AtomicInteger projectIdCounter = new AtomicInteger(1);
```

Comment:
You also can use UUID for id. Read about it. (Just for information, you dont neet to rewrite)